runtime/doc/diff.txt

```diff
@@ -354,13 +354,21 @@ The "1a2" item appends the line "bbb".
 The "4d4" item deletes the line "111".
 The "7c7" item replaces the line "GGG" with "ggg".
 
-When 'diffexpr' is not empty, Vim evaluates it to obtain a diff file in the
-format mentioned.  These variables are set to the file names used:
+When 'diffexpr' is a non-empty expresion, Vim evaluates it to obtain a diff file
+in the format mentioned.  These variables are set to the file names used:
 
 	v:fname_in		original file
 	v:fname_new		new version of the same file
 	v:fname_out		where to write the resulting diff file
 
+When 'diffexpr' is an expression prefixed with '->', Vim evaluates it to obtain
```

Comment:
Cool. I did this because I wasn't sure how to pass `bufnr_in` and `bufnr_new` as arguments to the ref-ed function, and how to retrieve `diff_chunks` value in return. I'll investigate into `call_func_retlist` :)